doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -630,6 +630,13 @@ Matter samples that used :ref:`zephyr:nrf-ieee802154-rpmsg-sample` or :ref:`mult
 
     DFU support for the nRF54L15 PDK is available only for the ``release`` build type.
 
+  * Support for Matter over Thread on the nRF54H20 DK with the ``nrf54h20dk/nrf54h20/cpuapp`` board target to the following Matter samples:
```

Comment:
```suggestion
  * Support for Matter over Thread on the ``nrf54h20dk/nrf54h20/cpuapp`` board target to the following Matter samples:
```

Reply:
Hmm previously we used the same naming for nRF54L15. @wiba-nordic what do you think?

Reply:
Seems we don't have any consistency on how this is expressed between the different teams in the NCS. We can stick with how it is now (it's the most information), and I'll bring this up with the docs team when we do final release notes review for the next release.